### PR TITLE
Report all missing namespaces at once

### PR DESCRIPTION
**Pull request by adutra**
Opening this to trigger a discussion around this idea. We would need to increment Nessie's spec to 2.1.3 I guess.

### CODE REVIEW

File: versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java [hunk 1]
```diff
@@ -62,16 +63,17 @@ protected AbstractNamespaceValidation(VersionStore store) {
...
-        ContentKey.of("ns2", "ns", "table"));
+    return Stream.of(ContentKey.of("ns", "table"), ContentKey.of("ns2", "ns", "table"));
   }
 
   @ParameterizedTest
   @MethodSource("contentKeys")
   void commitWithNonExistingNamespace(ContentKey key) throws Exception {
+
+    assumeTrue(
+        key.getElementCount() == 2 || !isNewStorageModel(),
```
Comment (dimas-b) [inline versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java:74]: this leaves only one parameter for this method, why bother with `@ParameterizedTest`?

**Review by dimas-b — COMMENTED** [review]
LGTM overall :)

**Comment by dimas-b** [comment]
+1 to bumping the spec version

File: versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java (hunk 1 above)
Comment (adutra) [reply versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java:74]: We are still testing the multiple missing namespaces case for the old model here.

Comment (dimas-b) [reply versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java:74]: `!isNewStorageModel()` makes sense, but do we have more one parameter?

Comment (dimas-b) [reply versioned/tests/src/main/java/org/projectnessie/versioned/tests/AbstractNamespaceValidation.java:74]: sorry, misinterpreted :facepalm:

**Review by dimas-b — APPROVED** [review]

**Pull request merged**